roles/setup_hammerdb/defaults/main.yml

```diff
@@ -6,8 +6,21 @@ hammerdb_filename: "HammerDB-{{ hammerdb_version }}-\
                     {{ ansible_distribution_major_version if \
                     ansible_os_family == 'RedHat'}}.tar.gz"
 hammerdb_url: "https://github.com/TPC-Council/HammerDB/releases/download/"
+hammerdb_user: "hammerdb"
```

Comment:
I would also add a `hammerdb_group: "hammerdb"` just to stay consistent with other roles.

Reply:
Will do.